releasenotes-builder/src/main/resources/com/github/checkstyle/templates/github_post.template

```diff
@@ -28,7 +28,6 @@ Bug fixes:
 <details>
 <summary>Other Changes:</summary>
 <#list notesMessages as message>
-<br/>

```

Comment:
After summary tag, we need BR tag.
Please attentively read issue description.

Reply:
Changes Done